db/schema.rb

```diff
@@ -20,16 +20,16 @@
     t.string   "name",              null: false
     t.boolean  "is_paid"
     t.boolean  "is_permanent_paid"
-    t.datetime "created_at"
-    t.datetime "updated_at"
+    t.datetime "created_at",        null: false
```

Comment:
Yeah. I don't know what causes it but I don't think it'll do any harm. I'm going to just shrug and move with it.